fineract-provider/src/main/java/org/apache/fineract/organisation/staff/domain/StaffRepository.java

```diff
@@ -27,10 +27,36 @@
 
     public final static String FIND_BY_OFFICE_QUERY = "select s from Staff s where s.id = :id AND s.office.id = :officeId";
 
+    public final static String IS_ANY_ACTIVE_ENTITY_ASSOCIATED_WITH_STAFF_QUERY = "select " +
```

Comment:
Instead of adding a native query in StaffRepository, add new method in StaffReadPlatformService and use JDBC template to run this query